CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 ## 3.76.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_certificate_manager_certificate` ([#9540](https://github.com/hashicorp/terraform-provider-google/pull/9540))
```

Comment:
These certificate_manager resources should be removed from the release if they haven't been already

Reply:
https://github.com/hashicorp/terraform-provider-google/commits/release-3.76.0

Reply:
looks good, they are out